packages/blocks/src/image-block/ledits/utils.ts

```diff
@@ -0,0 +1,132 @@
+export function createInlineIframe() {
+  const htmlContent = `<!DOCTYPE html>
```

Comment:
```ts
const htmlContent = html`<html></html>`
const rawString = String.raw(html.strings, ...html.values);
```

Use the `html` tagged template literal from Lit to highlight text in the editor. To get the original string, use `String.raw(html.strings, ...html.values)`.